src/core/public/public.api.md

```diff
@@ -166,24 +166,12 @@ export interface FatalErrorsSetup {
     get$: () => Rx.Observable<FatalErrorInfo>;
 }
 
-// @public
-export interface I18nSetup {
-    Context: ({ children }: {
-        children: default.ReactNode;
-    }) => JSX.Element;
-}
-
 // @public (undocumented)
-export type I18nStart = I18nSetup;
-
-// Warning: (ae-missing-release-tag) "IHttpService" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-// 
-// @public (undocumented)
-export interface IHttpService {
+export interface HttpServiceBase {
     // (undocumented)
     addLoadingCount(count$: Observable<number>): void;
     // (undocumented)
-    addToPath(path: string): string;
+    appendToBasePath(path: string): string;

```

Comment:
nit: this name is a bit misleading. It implies that it mutates some internal state rather than return a new string. I actually like the name we use in the legacy platform: `appendBasePath`

Reply:
Alternatively, you could namespace this function under `http.basePath.addtoPath`

Reply:
Hmm, actually I like it. Coming right up.

Reply:
One comment, it would be `prependBasePath`.

Reply:
Which closely matches the `fetch` option even more: `prependBasePath?: boolean;`

Reply:
Nice, I like that!